cypress/tests/data/60-content/VkarbasizaedSubmission.spec.js

```diff
@@ -53,10 +57,151 @@ describe('Data suite tests', function() {
+		cy.get('span:contains("' + issueTitle + '")').prev('a.show_extras').click();
+		cy.get('tr:contains("' + issueTitle + '")').next().contains('a', 'Publish Issue').click();
+		cy.get('input[id="sendIssueNotification"]').click();
+		cy.get('button[id^=submitFormButton]').click();
+		// check status = 3
+		cy.visit('index.php/publicknowledge/submissions');
+		cy.get('button[id="active-button"]').click();
+		cy.contains('View ' + familyName).click({force: true});
+		// check submission status
+		cy.get('span').should('have.class', 'pkpWorkflow__identificationStatus').contains("Published");
+		// check publication status
+		cy.get('button[id="publication-button"]').click();
+		cy.get('span').should('have.class', 'pkpPublication__status').contains("Published");
+		cy.contains('This version has been published and can not be edited.');
+		// the button "Unpublish" (connected with the publication)
+		// and the button "Create New Version" (connected with submission) exist
+		cy.get('div#publication button:contains("Unpublish")');
+		cy.get('div#publication button:contains("Create New Version")');
+	});
+
+	it('Remove submisison from TOC', function() {
+		cy.login('dbarnes');
+		cy.visit('index.php/publicknowledge/manageIssues#back');
+		cy.get('span:contains("' + issueTitle + '")').prev('a.show_extras').click();
+		cy.get('tr:contains("' + issueTitle + '")').next().contains('a', 'Edit').click();
+		cy.get('span:contains("' + submissionTitle + '")').prev('a.show_extras').click();
+		cy.get('tr:contains("' + submissionTitle + '")').next().contains('a', 'Remove').click();
+		cy.get('div:contains("Are you sure you wish to remove this article from the issue? The article will be available for scheduling in another issue.")');
+		cy.get('button:contains("OK")').click();
+		cy.get('span:contains("' + submissionTitle + '")').should('not.exist');
+		// check status = 1
+		cy.visit('index.php/publicknowledge/submissions');
+		cy.get('button[id="active-button"]').click();
+		cy.contains('View ' + familyName).click({force: true});
+		// check submission status
+		cy.get('span').should('not.have.class', 'pkpWorkflow__identificationStatus');
+		// check publication status
+		cy.get('button[id="publication-button"]').click();
+		cy.get('span').should('have.class', 'pkpPublication__status').contains("Unscheduled");
+		// the button "Schedule For Publication" exists
+		cy.get('div#publication button:contains("Schedule For Publication")');
+	});
 
+	it('Return back to the original state', function() {
+		cy.login('dbarnes');
 		// Publish in current issue
-		cy.publish('1', 'Vol. 1 No. 2 (2014)');
+		cy.visit('index.php/publicknowledge/submissions');
+		cy.get('button[id="active-button"]').click();
+		cy.contains('View ' +familyName).click({force: true});
+		cy.get('button[id="publication-button"]').click();
+		cy.get('button[id="issue-button"]').click();
+		cy.get('span:contains("This has not been scheduled for publication in an issue.")').next().click()
```

Comment:
I notice this is [where it's failing](https://travis-ci.com/github/pkp/ojs/jobs/472483101#L2922) and that's because I recently changed some of the text when a publication is assigned to an issue but not scheduled. https://github.com/pkp/pkp-lib/issues/6391. I think the new text at this state will be:

> This has been assigned to <a href=\"{$issueUrl}\">{$issueName}</a> but it has not been scheduled for publication.

Reply:
oh, strange, I've rebased with master... I'll double check! :+1:

Reply:
It was really recent! I think I just changed it on Thursday or Friday.